target/scripts/helper-functions.sh

```diff
@@ -187,9 +187,9 @@ function _extract_certs_from_acme
 }
 export -f _extract_certs_from_acme
 
-# Remove the `*.` prefix if it exists
+# Remove the `*.` prefix if it exists, else returns the input value
 function _strip_wildcard_prefix {
-  [[ "${1}" == "*."* ]] && echo "${1:2}"
+  [[ "${1}" == "*."* ]] && echo "${1:2}" || echo "${1}"
```

Comment:
More than surprised ShellCheck did not pick this up. IIRC in Bash, this is not an if-then-else. But I will look that up.

Reply:
The second `echo `is not executed, if the first `echo `fails. That's why you thought, it's not a real if-statement.

I think in that case, the shorthand is fine, as echo is unlikely to fail.

Reply:
I guess it's fine in this case, right.

Reply:
I've edited my comment above. It's the opposite of what I wrote 😖